src/overview/cli.md

```diff
@@ -87,3 +87,19 @@ Options:
  --no (-n)             Answer "no" to all prompts
  --shell (-s)          Launch the shell
 ```
+
+## Installing the CLI on Windows 10
+
+To install the CLI on a Windows 10, 64-bit machine, BASH needs to be installed. Otherwise, you will need to install a Virtual Machine running Linux. The Windows 10 Anniversary Update is needed for support for git and apt/apt-get.
```

Comment:
Theoretically, there are a lot of other options. The CLI could work with Git Bash, Cygwin, MinGW, and even (reportedly) PowerShell. So this should be phrased more like:

```
To install the CLI on Windows 10, we recommend using Bash.
```

and mention the Anniversary Update under the steps below